http/base/src/main/java/org/wildfly/security/http/HttpServerAuthenticationMechanismFactory.java

```diff
@@ -49,4 +49,7 @@
      */
     HttpServerAuthenticationMechanism createAuthenticationMechanism(String mechanismName, Map<String, ?> properties, CallbackHandler callbackHandler) throws HttpAuthenticationException;
 
+    default void shutdown() {
```

Comment:
Would be good to get some javadoc added here.

Reply:
Fixed